Engine/src/Engine/Scripts/MoveRightScript.h

```diff
@@ -0,0 +1,13 @@
+#include "Engine/Scripts/ScriptableBehavior.h"
```

Comment:
We can remove this from this PR, it was just an example script that doesn't really serve a purpose